tests/rptest/services/redpanda_cloud.py

```diff
@@ -144,97 +144,97 @@ def partitions_upper_limit(self):
     #    |        |        |   |         |          |   |      |     +- memory_per_broker
     #    |        |        |   |         |          |   |      |     |
     CloudTierName.AWS_1: AdvertisedTierConfig(
-         20*MiB,  60*MiB,  3,  512*MiB,  300*GiB,   20, 1000,  1500, 32*GiB
+         20*MiB,  60*MiB,  3,  64*MiB,  300*GiB,   20, 1000,  1500, 32*GiB
     ),
     CloudTierName.AWS_2: AdvertisedTierConfig(
-         50*MiB, 150*MiB,  3,  512*MiB,  500*GiB,   50, 2000,  3750, 64*GiB
+         50*MiB, 150*MiB,  3,  64*MiB,  500*GiB,   50, 2000,  3750, 64*GiB
     ),
     CloudTierName.AWS_3: AdvertisedTierConfig(
-        100*MiB, 200*MiB,  6,  512*MiB,  500*GiB,  100, 5000,  7500, 64*GiB
+        100*MiB, 200*MiB,  6,  64*MiB,  500*GiB,  100, 5000,  7500, 64*GiB
     ),
     CloudTierName.AWS_4: AdvertisedTierConfig(
-        200*MiB, 400*MiB,  6,    1*GiB, 1000*GiB,  100, 5000, 15000, 96*GiB
+        200*MiB, 400*MiB,  6,    128*MiB, 1000*GiB,  100, 5000, 15000, 96*GiB
     ),
     CloudTierName.AWS_5: AdvertisedTierConfig(
-        400*MiB, 800*MiB,  9,    1*GiB, 1000*GiB,  150, 7500, 30000, 96*GiB
+        400*MiB, 800*MiB,  9,    128*MiB, 1000*GiB,  150, 7500, 30000, 96*GiB
     ),
     CloudTierName.AWS_1_P5: AdvertisedTierConfig(
-         20*MiB,  60*MiB,  3,  512*MiB,  300*GiB,   20, 1000,  1500, 32*GiB
+         20*MiB,  60*MiB,  3,  64*MiB,  300*GiB,   20, 1000,  1500, 32*GiB
```

Comment:
curious to know how 64 MiB was chose? according to the config profile for this tier, the min is 16 MiB and max is 128 MiB

Reply:
Original was 512MB and 1GB, so the logic was half max is where 512 was